### DISCUSSION TITLE
Bulk editing appears broken

### DISCUSSION THREAD

**Discussion by mitchplze** [netbox-community/netbox#16548] [Help Wanted!] [answered]
Running two unrelated v4.0.5 environments, both in Docker. Separate hosts, separate domains, separate companies and users. Both are behind Caddy containers which handle the HTTPs, with simple reverse_proxy to the HTTP port of NetBox.

I'm unable to perform bulk edits/updates to anything. I've tried with devices, and with interface renames. When I click Apply, nothing happens.

This happens on both environments, whether I'm in Safari or Chrome. Console shows something that looks like broken JavaScript, but not sure.

Is this a bug, or am I doing something wrong? Was working fine until the 4.0.5 update. I've tried restarting all the containers as well.

![image](https://github.com/netbox-community/netbox/assets/25162574/49298667-0c01-4cd0-b8a4-5dcd61b02f0f)
![image](https://github.com/netbox-community/netbox/assets/25162574/d5cfc1dd-1f85-4cb9-bb22-96a438dba100)

**Comment by candlerb** [comment, marked as answer]
Do you have the experimental HTMX navigation turned on in your user preferences? Try turning it off.

![image](https://github.com/netbox-community/netbox/assets/44789/98db8df1-0ccc-4b9f-8eff-78eac83ead68)

Search for "HTMX" or "%22true%22" in github issues for more info.

**Comment by mitchplze** [comment]
I feel silly for having missed / forgotten I enabled that. Turning it off resolved the issue. Thanks very much!
